icicle/README.md

```diff
@@ -6,14 +6,7 @@
 mkdir -p build; cmake -S . -B build; cmake --build build; cd build && ctest; cd ..
 ```
 
-## Prerequisites on Ubuntu
-
-Before proceeding, make sure the following software installed:
-
-1. CMake at least version 3.18, which can be downloaded from [cmake.org](https://cmake.org/files/)
-   It is recommended to have the latest version installed.
-2. [CUDA Toolkit](https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=Ubuntu) version 12.0 or newer.
-3. GCC - version 9 or newer recommended.
+The command above will build ICICLE Core and run the ctest.
```

Comment:
Our cmake process changed a bit

`-DCURVE=<supported_curve>` where `supported_curve` is one of [these curves](https://github.com/ingonyama-zk/icicle/blob/main/icicle/CMakeLists.txt#L64) is needed for tests or building the lib

`-DBUILD_TESTS=ON` is required if you want to test

Reply:
thanks forgot this :)